src/ng/animate.js

```diff
@@ -369,14 +369,36 @@ var $AnimateProvider = ['$provide', /** @this */ function($provide) {
        * );
        * ```
        *
+       * <div class="alert alert-warning">
+       * **Note**: Generally, the events that are fired correspond 1:1 to `$animate` method names,
+       * e.g. {@link ng.$animate#addClass addClass} will fire `addClass`, and {@link ng.ngClass}
```

Comment:
Is this going to look a bit weird when rendered?

> ... e.g. `addClass` will fire `addClass`, ...

Shouldn't it be something like

> ... e.g. `$animate.addClass()` will fire `addClass`, ...

Reply:
The methods are not rendered in code-style, but blue - and I've added the parantheses to the methods names